modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA command line interface (CLI).
+
+[IMPORTANT]
+====
+You must ensure that your tag keys are not `aws`, `red-hat-managed`, `red-hat-clustertype`, or `Name`. In addition, you must not set a tag key that begins with `kubernetes.io/cluster/`. Your tag's key cannot be longer than 128 characters, while your tag's value cannot be longer than 256 characters. Red Hat reserves the right to add additional reserved tags in the future.
+====
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+* Create a machine pool with a custom tag by running the following command:
++
+--
+[source,terminal]
+----
+$ rosa create machinepools --cluster=<name> --replicas=<replica_count> \
+     --name <mp_name> --tags='<key> <value>,<key> <value>' \ <1>
+----
+<1> Replace `<key> <value>,<key> <value>` with a key and value for each tag.
+--
++
+.Example output
+[source,terminal]
+----
+$ rosa create machinepools --cluster=mycluster --replicas 2 --tags='tagkey1 tagvalue1,tagkey2 tagvaluev2'
+
+I: Checking available instance types for machine pool 'mp-1'
+I: Machine pool 'mp-1' created successfully on cluster 'mycluster'
+I: To view the machine pool details, run 'rosa describe machinepool --cluster mycluster --machinepool mp-1'
+I: To view all machine pools, run 'rosa list machinepools --cluster mycluster'
+----
+
+.Verification
+
+* Use the `describe` command to see the details of the machine pool with the tags, and verify that the tags are included for your machine pool in the output.:
++
+[source,terminal]
+----
+$ rosa describe machinepool --cluster=<cluster_name> <machine-pool-name>
```

Comment:
Nit - in the last user replaced value, there should be underscores instead of hyphens:
```suggestion
$ rosa describe machinepool --cluster=<cluster_name> <machine_pool_name>
```